client/src/features/modal/ModalCard.jsx

```diff
@@ -46,32 +51,34 @@ const ModalCard = ({ handleClose }) => {
         </button>
         <div className="w-4/6 mt-3 flex flex-col">
           <h2 className=" flex mb-1 border-solid border-b-2 border-black font-semibold">
-            <img className="w-8 pr-2" src={togetherLogo} alt="" /> The Title
+            <img className="w-8 pr-2" src={togetherLogo} alt="" /> {context.event.title}
           </h2>
           <div className="dateTime">
             <section className="flex m-3 gap-1 font-semibold">
               <GrCalendar className="mt-1" />
               <span className="">Day:</span>{" "}
-              <span>Thursday, November 11, 2022</span>
+              <span>{format(parseISO(context.event.startAt), 'M')}/{format(parseISO(context.event.startAt), 'd')}/{format(parseISO(context.event.startAt), 'y')}</span>
             </section>
 
             <section className="flex m-3 gap-1 font-semibold">
               <FaRegClock className="mt-1" />
-              <span className=" "> Starts: 11:11</span>
-              <span className="ml-9">Ends: 12:00</span>
+              <span className=" "> Starts: {format(parseISO(context.event.startAt), 'p')}</span>
+              <span className="ml-9">Ends: {format(parseISO(context.event.endAt), 'p')}</span>
```

Comment:
The formatToLocalTime utility function is working as expected, should consider using that.

```suggestion
 <span className=" "> Starts: {formatToLocalTime(context.event.startAt)}</span>
 <span className="ml-9">Ends: {formatToLocalTime(context.event.endAt)}</span>
```